README.md

```diff
@@ -52,17 +49,15 @@ This is a quick non-admin style logger that can log:
 #### Admin Loggers
 
 Admin Loggers are an embed style logger that will post realtime, detailed events to a chosen channel, these events are
-presented in a rich-embed. All triggered events contain an executor, in that will tell you who did what.
+presented in a rich-embed. All triggered events contain an executor, the executor is the member who triggered the event;
+for example, if i deleted your message in Discord, then i would be the executor
 
-for example, if a message is deleted, or a member is kicked or banned, or even if a channel was modified or created. It
-will query your servers event log and pull the person who did that action. so it will tell you who deleted the message,
-kicked or banned the user or deleted the channel.
+for example, if a message is deleted, kicked, banned, or even if a channel was modified or created. It will display who
+caused that action from the audit logs
```

Comment:
I don't think you can kick or ban a message 😄

Reply:
You could prob kick it but you might hurt its feelings

Reply:
> for example, if a message is deleted, kicked, banned, or even if a channel was modified or created. It will display who

yea...